linode/manager
# refactor: [M3-7440] - Update React Query Packages

## Description 📝

> [!important]
> This PR is getting merged into `feature/react-query-v4`
>
> Typecheck, e2e test, and unit tests are all expected to fail. The app will build, but will not work otherwise.
>
> Follow up tickets will slowly resolve the issues as part of this project

- This PR begins the task for updating React Query 🎉
- View our internal Epic for content (M3-7418)

## Changes  🔄
- Updates React Query to v4
- Replaces imports throughout the codebase (codemod used)

## How to test 🧪

- Verify the diff looks okay
- Verify the packages got updated
- Verify the imports have been replaced

## As an Author I have considered 🤔
